api/app/Models/User.php

```diff
@@ -16,25 +16,18 @@
  * @property string $email
  * @property string $first_name
  * @property string $last_name
- * @property string $password
+ * @property string $telephone
+ * @property string $preferred_lang
+ * @property Illuminate\Support\Carbon $created_at
+ * @property Illuminate\Support\Carbon $updated_at
  */
+
 class User extends Model
 // implements AuthenticatableContract, AuthorizableContract
```

Comment:
Is the commented code to be implemented later or can it be removed?

Reply:
I was assuming we would implement it later, but on second thought its unclear if we will even use these feature for our auth. I'll remove it - it can always be added in later.